AgendaUWP/AgendaUWP/App.xaml.cs

```diff
@@ -36,7 +36,7 @@ public App()
 
         protected override Task OnLaunchApplicationAsync(LaunchActivatedEventArgs args)
         {
-            NavigationService.Navigate(PageTokens.MainPage, null);
+            NavigationService.Navigate(PageTokens.EditContactPage, null);
```

Comment:
Why navigate directly to EditContactPage instead of MainPage?

Reply:
Ok! Done!